config-files/skin_fingertips.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE robot PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+
+<robot name="testSkin" portprefix="testSkin" xmlns:xi="http://www.w3.org/2001/XInclude" build="1">
+     <!--params>
+    <xi:include href="hardware/" />
+    </params>
+           
+    <!-- xmlns:xi="http://www.w3.org/2001/XInclude" name="fingertip" type="canBusSkin">
+        <param name="canbusDevice"> "socketcan" </param>
+        <param name="physDevice"> "socketcan" </param>
+        <param name="canDeviceNum"> 0 </param>
+        <param name="skinCanIds"> 10 </param>
+        <param name="period"> 10 </param>
+
+        <!-- 4C Message is the CMD_TACT_SETUP initialisation message. 
+             It is sent to all boards. ->
+        <param name="4C_Timer">         0x03 </param>
+        <param name="4C_CDCOffsetL">    0x00 </param>
+        <param name="4C_CDCOffsetH">    0x20 </param>
+        <param name="4C_TimeL">         0x00 </param>
+        <param name="4C_TimeH">         0x35 </param>
+
+        <!-- 4E Message is the CMD_TACT_SETUP2 initialisation message. 
+             So far it is sent only to the hand board (14) to double the gain for the new (V2) fingertips. ->
+        <param name="4E_Shift">     0x00 </param>
+        <param name="4E_Shift3_1">  0x22 </param>
+        <param name="4E_NoLoad">    0xF0 </param>
+        <param name="4E_Param">     0x00 </param>
+        <param name="4E_EnaL">      0xFF </param>
+        <param name="4E_EnaH">      0xFF </param->
+    </device-->  
```

Comment:
Can you remove these commented lines or provide a reason as to why we should keep them?

Reply:
addressed in 047103a7bdc42c7d70c0c16f8bfc92834809a21d